contentcuration/contentcuration/frontend/shared/urls.js

```diff
@@ -0,0 +1,7 @@
+const urls = {};
+
+if (globalThis.Urls) {
```

Comment:
Should this be just `global`?

Reply:
No, this is intended. It's a way to refer to either `window` or `self` depending on non-worker or worker context:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

Reply:
(Although looking at the support, might be better to try another strategy)

Reply:
Have updated to use a shim instead.